app/views/layouts/_analytics.html.erb

```diff
@@ -1,17 +1,20 @@
 <!-- Google Analytics -->
 <script>
+/* global ga, window */
+window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments); };
 
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+var dimensions = {
+  version: {
+    dimension: 'dimension2',
+    payload: 'v1'
+  }
+};
 
 ga('create', 'UA-86101042-3', 'auto');
 ga('set', 'anonymizeIp', true);
 ga('set', 'displayFeaturesTask', null);
-ga('set', 'transport', 'beacon');
```

Comment:
is there a downside to setting this given that GA will fallback to a supported method as per: https://developers.google.com/analytics/devguides/collection/analyticsjs/sending-hits#specifying_different_transport_mechanisms does it break IE?

Reply:
Didn't realise Analytics was sensible enough to provide a fallback - should've checked the reports to see if IE was registering first. Thanks for the catch! Fixed now.